hasura/migrations/default/1705187600730_alter_table_app_drop_metadata/up.sql

```diff
@@ -0,0 +1,11 @@
+alter table "public"."app"
```

Comment:
let's move this to a separate PR as we'll need to merge this PR before the initial production deployment